sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/Microsoft.Azure.HDInsight.Job.csproj

```diff
@@ -1,23 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <!-- Please do not move/edit code below this line -->
-    <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-    <!-- Please do not move/edit code below this line -->
-
     <PropertyGroup>
         <PackageId>Microsoft.Azure.HDInsight.Job</PackageId>
         <Description>Azure HDInsight Job SDK Library</Description>
         <AssemblyName>Microsoft.Azure.HDInsight.Job</AssemblyName>
         <Version>3.0.0-preview</Version>
         <PackageTags>Microsoft Azure HDInsight Job;HDInsight;HDInsight Job</PackageTags>
+        <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
```

Comment:
Why did you need to set these? If you are converting this to be like the other data-plane libraries you should instead removed the AssemblyInfo.cs file and let the build pipeline generate it now. Please verify that they same versions exist in the generated version as the one that was checked in.

Reply:
For reference you can see the changes I made when I switch KeyVault over to using the data-plane targets. https://github.com/Azure/azure-sdk-for-net/pull/5149